pull/github.go

```diff
@@ -356,7 +356,7 @@ func (ghc *GitHubContext) loadCollaboratorData() error {
 					Permission string  `graphql:"permission"`
 					Node       v4Actor `graphql:"node"`
 				}
-			} `graphql:"collaborators(first: 100, after: $collaboratorCursor, affiliation: DIRECT)"`
```

Comment:
I need to rethink the flow "user is an org admin, but we don't want to ask them for review". Perhaps I can use https://developer.github.com/v4/object/permissionsource/, or run another check later

Reply:
Using `PermissonSource` and `PermissionGranter` feels like it would work and should avoid additional queries.

Reply:
I've opened #123 to hopefully deal with org owners as a separate feature. I think it would be more friendly to merge this, and iterate there for the "best" solution if we can't use the v4 api.